### DISCUSSION TITLE
How to use externalAuthenticationURL?

### DISCUSSION THREAD

**Discussion by gj357d** [bluenviron/mediamtx#2001] [Questions]
Hi everybody,

I would need authentication for multiple users. I am not able and not sure if it is possible to create more users (5 users) in .yml. I decided to take a look at "externalAuthenticationURL" I have some API experience on HomeAssistant, but I can't move on at the moment.

- Is it possible to set up multiple users for one stream in .yml?
- Which API server is best to use and how to configure it? (for Windows 7 as localhost)
- Can you please give more examples of what to include in:
  {
    "ip": "ip",
    "user": "user",
    "password": "password",
    "path": "path",
    "action": "read|publish"
    "query": "url's raw query"
  }
?

Well thank you.

**Comment by xNineteenUK** [comment]
i was thinking the same but changing it instead of using user/password use a stream key instead, iv not get to the point of learning how, json works inside it all just yet, but if there is still no response here, i might have a look at it as there is a pritty poor api

[a comment by an automated account is not shown]
